src/shared/varlink.c

```diff
@@ -462,7 +481,44 @@ static int varlink_write(Varlink *v) {
          * Use a local variable to help gcc figure out that we set 'n' in all cases. */
         bool prefer_write = v->prefer_read_write;
         if (!prefer_write) {
-                n = send(v->fd, v->output_buffer + v->output_buffer_index, v->output_buffer_size, MSG_DONTWAIT|MSG_NOSIGNAL);
+                struct msghdr msgh = {0};
```

Comment:
we rely on the gcc extension that allows us to write this simply as `… = {};`, i.e. without the `0`